src/styles/cards.scss

```
@@ -0,0 +1,143 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(4,200px);
+  grid-gap: 46px 48px;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 72px;
+}
+
+.card {
+  border: 1px solid $border-color;
+  width: 200px;
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+  background-color: $white-color;
+  border-radius: 5px;
+  justify-content: space-between;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__product-name {
+    color: $title-color;
+  }
+
+  &__image {
+    margin: 19px;
+    margin-top: 32px;
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    padding: 21px 16px 0;
+    color: $black-color;
+  }
+
+  &__product-code {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 16px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    color: $gray-color;
+  }
+
+  &__stars {
+    margin: 0;
+    padding: 0;
+    height: 16px;
+    border-radius: 0.5px;
+    align-content: center;
+  }
+
+  &__stars-img {
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    display: inline-block;
+    background-image: url(images/star.svg);
```

Combine with [`background`](https://developer.mozilla.org/en-US/docs/Web/CSS/background) property

```suggestion
    display: inline-block;

    background: ..... ;
```